### PR TITLE
Include DISTRIBUTION_CONTAINER

### PR DESCRIPTION
**Pull request by grisu48**
test_all does not perform tests on the distribution container because it was not defined in the ALL_CONTAINERS. This commit adds it to the CONTAINERS_WITHOUT_ZYPPER group so that it is included in the `all` test run.

* Related issue: https://progress.opensuse.org/issues/129208
* Verification run: https://duck-norris.qe.suse.de/tests/12891

### CODE REVIEW

**Review by jlausuch — DISMISSED** [review]

**Review by dirkmueller — CHANGES_REQUESTED** [review]
tests/test_metadata.py : IMAGES_AND_NAMES needs to be updated as well.

File: tests/test_metadata.py [hunk 1]
```diff
@@ -163,6 +163,9 @@ def _get_container_label_prefix(
         ]
         if LOCALHOST.system_info.arch == "x86_64"
         else []
+        + [
```
Comment (dcermak) [inline tests/test_metadata.py:166]: No, this is the wrong place. What you just did is adding the distribution container on non-x86_64. Please add it after postgres

**Review by dcermak — CHANGES_REQUESTED** [review]

File: tests/test_metadata.py (hunk 1 above)
Comment (grisu48) [reply tests/test_metadata.py]: Done.

**Review by dcermak — CHANGES_REQUESTED** [review]
`tox -e format` and rebase on main please.

File: tests/test_metadata.py [hunk 2]
```diff
@@ -125,6 +126,9 @@ def _get_container_label_prefix(
         (pg_container, "postgres", ImageType.APPLICATION)
         for pg_container in POSTGRESQL_CONTAINERS
     ]
+    + [
+        (DISTRIBUTION_CONTAINER, "distribution", ImageType.APPLICATION),
```
Comment (dcermak) [inline tests/test_metadata.py:130]: The image is called registry.
```suggestion
        (DISTRIBUTION_CONTAINER, "registry", ImageType.APPLICATION),
```

**Review by dcermak — CHANGES_REQUESTED** [review]

File: tests/test_metadata.py (hunk 2 above)
Comment (grisu48) [reply tests/test_metadata.py:130]: Yes, I check the changes first locally before waiting for the slow CI.

**Comment by grisu48** [comment]
CI says no: https://github.com/SUSE/BCI-tests/actions/runs/4978540558/jobs/8908996272?pr=252

File: tests/test_metadata.py [hunk 3]
```diff
@@ -240,10 +244,12 @@ def test_general_labels(
         if container_type != ImageType.APPLICATION:
             assert "BCI" in labels[f"{prefix}.title"]
 
-        assert (
-            "based on the SLE Base Container Image."
-            in labels[f"{prefix}.description"]
-        )
+        # distribution images do not hold this label
```
Comment (dcermak) [inline tests/test_metadata.py:247]: nitpick: this is not correct, the label exists, just does not contain the sentence "based on the SLE Base Container Image."

Comment (grisu48) [reply tests/test_metadata.py]: amended. let's wait for the CI.

**Review by dcermak — APPROVED** [review]
LGTM, thanks for your patience :pray:!

**Comment by dcermak** [comment]
> The code is black formatted, CI is failing due to 503.

Oh Azure is just on :fire:, nothing to see here, carry on…

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**